vhd/packer/install-dependencies.sh

```diff
@@ -367,8 +367,8 @@ echo "  - busybox" >> ${VHD_LOGS_FILEPATH}
 K8S_VERSIONS="
 1.19.1
 1.19.0
+1.18.9
 1.18.8
-1.18.6
```

Comment:
cc @Michael-Sinz

The next (v0.57.0) release of aks-engine won't include 1.18.6 in the VHD, nor support new cluster creates using v1.18.6